docker-images/grafana/grafana_setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+mkdir -p /var/lib/grafana/dashboards
+cd /var/lib/grafana/dashboards
+curl "https://grafana.com/api/dashboards/1860/revisions/23/download" > "node_exporter_full.json"
+curl -O "https://raw.githubusercontent.com/k32/grafana-dashboards/master/app_top.json"
+curl -O "https://raw.githubusercontent.com/k32/grafana-dashboards/master/emqx_exporter.json"
+curl -O "https://raw.githubusercontent.com/k32/grafana-dashboards/master/node_exporter.json"
+curl -O "https://raw.githubusercontent.com/k32/grafana-dashboards/master/node_exporter_full.json"
```

Comment:
Q: won't this line overwrite the output of the [previous one](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#rolling-update-deployment) with the same file name?

Reply:
No, not with curl -O flag.